README.md

```diff
@@ -16,6 +16,295 @@ K8ssandra 1.x is configured, packaged, and deployed via Helm charts. Those Helm
 
 K8ssandra 2.x will be based on the this operator.
 
+One of the primary features of this operator is multi-cluster support which will facilitate multi-region Cassandra clusters.
+
+# Installing the operator
+There are a couple of options for installing the operator - build from source or remote installs via kustomize.
+
+**Note:** There are plans to add a Helm chart as well (TODO create and reference ticket here)
+
+**Note:** This section focuses on a single cluster install. See the multi-cluster section below for details on how to configure the operator for a multi-cluster install.
+
+## Remote Install
+Kustomize supports building resources with remote URLs. See this Kustomize [doc](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md) for background.
+
+This section provides some examples that demonstrate how to configure the operator installation via Kustomize.
+
+You need to Kustomize 4.0.5 or later installed. See [here](https://kubectl.docs.kubernetes.io/installation/kustomize/) for installation options. 
+
+Recent versions of `kubectl` include Kustomize. It is executed using the `-k` option. I prefer to install Kustomize and use the `kustomize` binary as I have found in the past that the one embedded with `kubectl` can be several versions behind and behave differently  than what is described in the Kustomize docs.
+
+### Default Install
+First, create a kustomization directory that builds from the `main` branch:
+
+```sh
+K8SSANDRA_OPERATOR_HOME=$(mktemp -d)
+cat <<EOF >$K8SSANDRA_OPERATOR_HOME/kustomization.yaml
+resources:
+- github.com/k8ssandra/k8ssandra-operator/config/default?ref=main
+EOF
+```
+
+Now install the operator:
+
+```
+kustomize build $K8SSANDRA_OPERATOR_HOME | kubectl apply -f -
+```
+This installs the operator in the `default` namespace.
+
+If you just want to generate the manifests then run:
+
+```
+kustomize build $K8SSANDRA_OPERATOR_HOME
```

Comment:
I get an error:
```
unable to recognize "STDIN": no matches for kind "Certificate" in version "cert-manager.io/v1"
unable to recognize "STDIN": no matches for kind "Issuer" in version "cert-manager.io/v1"
```

Reply:
I've re-applied a couple of times because sometimes the CRDs take a moment to materialise in the API server, but still getting the same problem a few minutes later. 

Are we adding the cert-manager CRDs as part of the kustomization bundle? It looks like we are not, from [here](https://github.com/k8ssandra/k8ssandra-operator/blob/b625d7681b33c0e64154147956b508c3259d18f3/config/default/kustomization.yaml#L28).

Reply:
I need to update the doc for cert-manager.

Basic kustomize support [here](https://github.com/k8ssandra/k8ssandra-operator/tree/main/config/cert-manager).